### ISSUE TITLE
Is there a way to run the doxygen CLI via an npm script?

### ISSUE THREAD

**Issue by kruncher** [EruantalonJS/node-doxygen#13]
If I use the following:
```sh
npm install --save doxygen
```

Is there a way to add a script to my project's `package.json` file:
```sh
{
  "scripts": {
    "build-docs": "doxygen ...params..."
  }
  ...
}
```

**Comment by EruantalonJS** [comment]
currently not, there is no way to pass arguments to node-doxygen from the command line.

This does seem like a nice addition so I'm adding it to the backlog and I will let you know once this is possible

**Comment by EruantalonJS** [comment]
This was implemented in version 0.3.0

The way to use it in your project would be:

` "scripts": {
    "install-version": "doxygen --download",
    "create-config": "doxygen --config",
    "build-docs": "doxygen --docs"
  }`

For more info about the CLI options you can refer to the readme in the project
